internal/modulecontext/module_context.go

```diff
@@ -20,6 +20,11 @@ import (
 // It is used for definitions of mock verbs as well as real implementations of verbs to directly execute
 type Verb func(ctx context.Context, req any) (resp any, err error)
 
+// Map is a function that takes an object and returns an object of a potentially different type but is not constrained by input/output type like ftl.Map
```

Comment:
There shouldn't be any changes to the `ModuleContext`...

Reply:
done